scalafmt-tests/src/test/resources/align/AlignByRightName.stat

```diff
@@ -71,6 +71,12 @@ libraryDependencies ++= Seq(
   "com.geirsson1111111" %%% "metaconfig-core" % metaconfigV,
   "org.scalacheck"       %% "scalacheck"      % scalacheckV
 )
+<<< sbt := operator
```

Comment:
@kitbellew So I tried adding this test and I am getting the following test error

```
==> X org.scalafmt.FormatTests.align/AlignByRightName.stat:74: sbt := operator                       |  0.01s munit.FailException: /home/mdedetrich/github/scalafmt/scalafmt-tests/src/test/resources/align/AlignByRightName.stat:74
73:)
74:<<< sbt := operator
75:scalaVersion in ThisBuild := "2.13.6"
test does not parse
scalaVersion in ThisBuild := "2.13.6"
version in ThisBuild := "0.1.0-SNAPSHOT"
^
/home/mdedetrich/github/scalafmt/scalafmt-tests/src/test/resources/align/AlignByRightName.stat:2: error: end of file expected but identifier found
version in ThisBuild := "0.1.0-SNAPSHOT"
^

scalaVersion in ThisBuild := "2.13.6"
version in ThisBuild := "0.1.0-SNAPSHOT"
    at munit.Assertions.fail(Assertions.scala:251)
    at munit.Assertions.fail$(Assertions.scala:244)
    at munit.FunSuite.fail(FunSuite.scala:11)
    at org.scalafmt.util.CanRunTests.$anonfun$runTest$2(CanRunTests.scala:22)
Caused by: /home/mdedetrich/github/scalafmt/scalafmt-tests/src/test/resources/align/AlignByRightName.stat:2: error: end of file expected but identifier found
version in ThisBuild := "0.1.0-SNAPSHOT"
^
    at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
    at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
    at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
    at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
    at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
    at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
    at scala.meta.internal.parsers.ScalametaParser.syntaxErrorExpected(ScalametaParser.scala:898)
    at scala.meta.internal.parsers.ScalametaParser.accept(ScalametaParser.scala:904)
    at scala.meta.internal.parsers.ScalametaParser.skipStatementSeparators$1(ScalametaParser.scala:4917)
    at scala.meta.internal.parsers.ScalametaParser.entrypointStat(ScalametaParser.scala:4923)
    at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseStat$2(ScalametaParser.scala:64)
    at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:53)
    at scala.meta.internal.parsers.ScalametaParser.parseStat(ScalametaParser.scala:64)
    at scala.meta.parsers.Parse$.$anonfun$parseStat$1(Parse.scala:14)
    at scala.meta.internal.parsers.ScalametaParser$$anon$261.apply(ScalametaParser.scala:5198)
    at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
    at org.scalafmt.config.ScalafmtRunner.parse(ScalafmtRunner.scala:39)
    at org.scalafmt.Scalafmt$.doFormatOne(Scalafmt.scala:121)
    at org.scalafmt.Scalafmt$.doFormat(Scalafmt.scala:102)
    at org.scalafmt.Scalafmt$.formatCode(Scalafmt.scala:71)
    at org.scalafmt.FormatTests.run(FormatTests.scala:43)
    at org.scalafmt.FormatTests.$anonfun$new$3(FormatTests.scala:32)
    at org.scalafmt.FormatTests.$anonfun$new$3$adapted(FormatTests.scala:32)
    at org.scalafmt.util.CanRunTests.$anonfun$runTest$2(CanRunTests.scala:19)
```

Do you have any idea what could be causing this? Its complaining about an `EOF` operator but there shouldn't be on there?

Reply:
What if you add:
```
runner.parser = source
===
scalaVersion in ThisBuild := "2.13.6"
version in ThisBuild := "0.1.0-SNAPSHOT"
>>>
scalaVersion in ThisBuild := "2.13.6"
version in ThisBuild      := "0.1.0-SNAPSHOT"
```

Files ending with `.stat` should only parse a single statement, but we can work around it this way I think.

Reply:
Figured it out, you have to enclose multine scala tests inside of `{` and `}`